### DISCUSSION TITLE
right click + scroll = next/prev image

### DISCUSSION THREAD

**Discussion by tidehunter1** [sylikc/jpegview#102] [Ideas]
Hold down right click and mouse scroll to go next/previous image
Normally used to switch tabs in Sublime Text or Vivaldi, figured the same functionality could be used to switch images.

**Comment by sylikc** [comment]
that's an interesting gesture... I haven't ever encountered that before in my apps... I'll see about it if it can be supported as an option

**Comment by pureby** [comment]
@tidehunter1 You can easily do this with AHK (AutoHotKey). Just create this script:
```
#IfWinActive ahk_exe JPEGView.exe
  ~RButton & WheelUp::Send {Left}
  ~RButton & WheelDown::Send {Right}
#IfWinActive
return
```
@sylikc Yes, this is a very cool gesture. Vivaldi browser uses it for tab scrolling. And I use it as Alt+Tab alternative in Windows (using AHK). Very handy ;-)

**Reply by sylikc** [reply]
So, the default WHEELUP/WHEELDOWN is to zoom... but, you can change wheelup/down to NEXT/LAST... and then change zoom to CTRL+wheelup/down.  There's a setting in the INI to control that.
